chain_of_trust_all-unix-style.go

```diff
@@ -12,13 +12,19 @@ import (
 )
 
 func (cot *ChainOfTrustTaskFeature) ensureTaskUserCantReadPrivateCotKey() error {
-	c, err := process.NewCommand([]string{"/bin/cat", config.SigningKeyLocation}, cwd, cot.task.EnvVars())
-	if err != nil {
-		panic(fmt.Errorf("SERIOUS BUG: Could not create command (not even trying to execute it yet) to cat private chain of trust key - %v", err))
+	signingKeyPaths := [2]string{
+		config.OpenPGPSigningKeyLocation,
+		config.Ed25519SigningKeyLocation,
 	}
-	r := c.Execute()
-	if !r.Failed() {
-		return fmt.Errorf(ChainOfTrustKeyNotSecureMessage)
+	for _, path := range signingKeyPaths {
+		c, err := process.NewCommand([]string{"/bin/cat", path}, cwd, cot.task.EnvVars())
+		if err != nil {
+			panic(fmt.Errorf("SERIOUS BUG: Could not create command (not even trying to execute it yet) to cat private chain of trust key - %v", err))
```

Comment:
nit - might be nice to add `path` variable to the panic message to show which one it failed on